api/api/specs/jobs.yaml

```diff
@@ -419,6 +419,10 @@ components:
           type: string
         resources:
           $ref: "#/components/schemas/EnsemblingResources"
+        environment_variables:
+          type: object
+          additionalProperties:
+            type: string
```

Comment:
Can we make it consistent (in terms of property name and schema definition) with `routers.yaml#/components/schemas/EnvVar`? 
As regarding schema definition, I'm not sure which one I prefer – `object` with `additionalProperties` or `array` with `EnvVar` items, so I'm open to discussion. Probably, the best would be to stick to `array[EnvVar]` (so it doesn't affect schema of routers) as a type persisted in the database (it's also consistent with how k8s deals with env vars), but allow user to set job's variables as dictionary via turing-sdk.

Reply:
Ok, did not realise this was used somewhere else as well.